oauth/v1/types.go

```diff
@@ -117,6 +120,10 @@ type OAuthClient struct {
 	// AccessTokenMaxAgeSeconds overrides the default access token max age for tokens granted to this client.
 	// 0 means no expiration.
 	AccessTokenMaxAgeSeconds *int32 `json:"accessTokenMaxAgeSeconds,omitempty" protobuf:"varint,8,opt,name=accessTokenMaxAgeSeconds"`
+
+	// AccessTokenTimeoutSeconds overrides the default token timeout for tokens
+	// granted to this client. 0 means no timeout.
+	AccessTokenTimeoutSeconds *int32 `json:"accessTokenTimeoutSeconds,omitempty" protobuf:"varint,9,opt,name=accessTokenTimeoutSeconds"`
```

Comment:
We use `Expiration` across most of the API.  `AccessTokenExpirationSeconds` would be more appropriate.

Reply:
I see, this is your inactivity timeout.

`AccessTokenInactivityTimeoutSeconds`

Reply:
Better Godoc here.  explain what it actually does.

Reply:
I can change the name, but the doc is basically the same as AccessTokenMaxAgeSeconds for consistency.
Also not sure what else there is to explain, it says it overrides the default token timeout for tokens, which is what it does ...

Reply:
Sure, but give an example.  This is directly surfaced into our public user facing documentation for the API, so some context about how the feature works is expected.  Doesn't have to be more than a paragraph, but we try to describe the feature in terms an API consumer would understand - what its for, how it works, etc.